source/memory.tex

```diff
@@ -2549,8 +2549,8 @@
 
 \pnum
 \ensures
-If \tcode{this != addressof(u)},
-\tcode{u.get() == nullptr},
+If \tcode{this != addressof(u)} is \tcode{true},
```

Comment:
Perhaps this (and similar condition below) should be inverted to `\tcod{this == addressof(u)} is \tcode{false}`.
I'm not sure which way is more readable

Reply:
I wonder whether we can simply say "`this == addressof(u) || u.get() == nullptr` is `true`".